addons/medical_treatment/functions/fnc_ivBagLocal.sqf

```diff
@@ -37,11 +37,33 @@ private _partIndex = ALL_BODY_PARTS find toLowerANSI _bodyPart;
 private _defaultConfig = configFile >> QUOTE(ADDON) >> "IV";
 private _ivConfig = _defaultConfig >> _treatment;
 
-private _volume   = GET_NUMBER(_ivConfig >> "volume",getNumber (_defaultConfig >> "volume"));
-private _type     = GET_STRING(_ivConfig >> "type",getText (_defaultConfig >> "type"));
-private _rateCoef = GET_NUMBER(_ivConfig >> "rateCoef",getNumber (_defaultConfig >> "rateCoef"));
+private _volume     = GET_NUMBER(_ivConfig >> "volume",getNumber (_defaultConfig >> "volume"));
+private _type       = GET_STRING(_ivConfig >> "type",getText (_defaultConfig >> "type"));
+private _rateCoef   = GET_NUMBER(_ivConfig >> "rateCoef",getNumber (_defaultConfig >> "rateCoef"));
+private _guiMessage = GET_STRING(_ivConfig >> "gui_message",getText (_defaultConfig >> "gui_message"));
+private _ratio      = GET_ARRAY(_ivConfig >> "ratio",getArray (_defaultConfig >> "ratio"));
 
 // Add IV bag to patient's ivBags array
 private _ivBags = _patient getVariable [QEGVAR(medical,ivBags), []];
-_ivBags pushBack [_volume, _type, _partIndex, _treatment, _rateCoef, _item];
+
+// Simulate mixed fluid bags by using multiple smaller ivs. rateCoef is lowered to compensate
+if (count _ratio > 2) then {
+    private _totalRatio = 0;
+
+    // Add an iv bag for each item in _ratio
+    {
+        if (_x isEqualType 1) then { continue }; // Skip over the numbers directly
+
+        private _ratioCoef = _ratio select (_forEachIndex + 1);
+        _totalRatio = _totalRatio + _ratioCoef;
+        _ivBags pushBack [_volume * _ratioCoef, _x, _partIndex, _treatment, _rateCoef * _ratioCoef, _item, _guiMessage];
```

Comment:
This does the localization on the patient machine and then broadcasts that value
Could use `getTextRaw` but I still don't like syncing a big string every time the bags update

Reply:
changed in addb928 if that looks ok?